lib/main_common.pm

```diff
@@ -3240,4 +3241,12 @@ sub load_kernel_baremetal_tests {
     loadtest "kernel/build_git_kernel" if get_var('KERNEL_GIT_TREE');
 }
 
+sub load_pynfs_tests {
+    if (check_var('PYNFS', 'installation')) {
```

Comment:
Please help to remove this if structure. The line `if (check_var('PYNFS', 'installation')) {}` is not suppose to use in previous PR, my bad.

Reply:
Removed :)

Reply:
Thanks!